extractor/src/main/scala/com/digitalasset/extractor/writers/postgresql/Queries.scala

```diff
@@ -288,9 +288,12 @@ object Queries {
             "?::jsonb",
             toJsonString(v)
           )
+
         case e @ V.ValueEnum(_, _) =>
-          // FixMe (RH) https://github.com/digital-asset/daml/issues/105
-          throw new NotImplementedError("Enum types not supported")
+          Fragment(
+            "?::jsonb",
+            toJsonString(e)
+          )
```

Comment:
We should copy `ValueText`'s semantics instead, not records' or variants'.

`MultiTableDataFormat#mapSQLType` needs a case under `TypeCon`, which means that the premise of `GenTypeOps#fat` (that the SQL type of an `iface.Type` may be determined given no other context) is now false, and `FullyAppliedType.TypeCon` needs to be extended with the information of whether or not the type is an enum.